/ground_removal_node:
  ros__parameters:
    min_dist: 3.0
    max_dist: 120.0
    num_bin: 120
    num_channel: 80
    min_height: -2.0
    max_height: -0.4
    sensor_height: 2.0
    hdiff: 0.4

/cluster_object_node:
  ros__parameters:
    num_grid: 200
    roi_m: 30.0
    kernel_size: 3
    pic_scale: 30.0
    lslope_dist: 3.0
    lnum_points: 300
    rnd_points: 80
    sensor_height: 1.73
    thr_height_min: 1.0
    thr_height_max: 2.6
    thr_width_min: 0.25
    thr_width_max: 3.5
    thr_len_min: 0.5
    thr_len_max: 14.0
    thr_area_max: 20.0
    tpt_per_m3: 8.0
    min_len_ratio: 3.0
    thr_ratio_min: 1.3
    thr_ratio_max: 5.0
